pom.xml

```diff
@@ -61,6 +61,8 @@
                     <artifactSet>
                         <includes>
                             <include>com.github.CodingAir:CodingAPI</include>
+                            <include>com.zaxxer:HikariCP</include>
```

Comment:
MySQL should work. Even without included repo.

Sorry if my last change request was not 100% clear. I don’t want to increase my jar with unnecessary libraries. There are available MySQL libraries and you should better use them instead of external ones.

Hope this is ok for you!

Reply:
Tbh I fail to see how I can know which libraries are already on the class path?
And isn't it dependents on the spigot version if MySQL is already on the class path. Is there somewhere a list of spigot dependencies? As far as I know basic mc server doesn't use mysql

Reply:
Yeah as I expected I am getting java.lang.NoClassDefFoundError errors.
Could you tell me which library is already present on the classpath I can use in a clean spigot installation?

I could not find any information on this online

Reply:
You can just use the default java.sql classes. It won’t allow hikari pools but you will be able to use default MySQL.

Reply:
I also noticed craftbukkit places an sqlite dependency on the class path so I might as well implement both databases?

Reply:
OK added sqlite and marked the dependencies as provided.